examples/electron_phonon/run_electron_phonon.py

```diff
@@ -0,0 +1,24 @@
+from pyvotca import Molecule
```

Comment:
I think that we can make the examples run as tests. I do that if you agree

Reply:
Good idea!